### PR TITLE
jpegtran: add page

### PR DESCRIPTION
**Pull request by ev-john**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Page requested #4641

### CODE REVIEW

File: pages/linux/jpegtran.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# jpegtran
+
+> Perform lossless transformation of JPEG files.
+> More information: <https://linux.die.net/man/1/jpegtran>.
+
+- Mirror image horizontally or vertically:
+
+`jpegtran -flip {{horizontal|vertical}} {{/path/to/image.jpg}}`
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:8]:
```suggestion
`jpegtran -flip {{horizontal|vertical}} {{path/to/image.jpg}} > {{path/to/output.jpg}}`
```

File: pages/linux/jpegtran.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# jpegtran
+
+> Perform lossless transformation of JPEG files.
+> More information: <https://linux.die.net/man/1/jpegtran>.
+
+- Mirror image horizontally or vertically:
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:6]:
```suggestion
- Mirror an image horizontally or vertically:
```

File: pages/linux/jpegtran.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`jpegtran -flip {{horizontal|vertical}} {{/path/to/image.jpg}}`
+
+- Rotate image 90, 180 or 270 degrees clockwise:
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:10]:
```suggestion
- Rotate an image 90, 180 or 270 degrees clockwise:
```

File: pages/linux/jpegtran.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`jpegtran -rotate {{90|180|270}} {{/path/to/image.jpg}}`
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:12]:
```suggestion
`jpegtran -rotate {{90|180|270}} {{path/to/image.jpg}} > {{path/to/output.jpg}}`
```

File: pages/linux/jpegtran.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Transpose the image across the upper left to lower right axis:
+
+`jpegtran -transpose {{/path/to/image.jpg}}`
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:16]:
```suggestion
`jpegtran -transpose {{path/to/image.jpg}} > {{path/to/output.jpg}}`
```

File: pages/linux/jpegtran.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Transverse the image across the upper right to lower left axis:
+
+`jpegtran -transverse {{/path/to/image.jpg}}`
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:20]:
```suggestion
`jpegtran -transverse {{path/to/image.jpg}} > {{path/to/output.jpg}}`
```

File: pages/linux/jpegtran.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Convert the image to grayscale:
+
+`jpegtran -grayscale {{/path/to/image.jpg}}`
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:24]:
```suggestion
`jpegtran -grayscale {{path/to/image.jpg}} > {{path/to/output.jpg}}`
```

File: pages/linux/jpegtran.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Crop the image to a rectangular region of width `W` and height `H`, starting at point `X` and `Y`:
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:26]: We should say which is the origin (point 0,0) of the coordinate system. Maybe this is better:
```suggestion
- Crop the image to a rectangular region of width `W` and height `H`, starting at point `X` and `Y` from the upper left corner:
```

File: pages/linux/jpegtran.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`jpegtran -crop WxH+X+Y {{/path/to/image.jpg}}`
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:28]: The variables should be between `{}`:
```suggestion
`jpegtran -crop {{W}}x{{H}}+{{X}}+{{Y}} {{path/to/image.jpg}} > {{path/to/output.jpg}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @ev-john, thanks for your submission. I left some comments.

We need to add an example for `-outfile` because in a command concatenation with `&&`, the user can't use the `stdout` and `-outfile` is easier.

Maybe something like this (also saying the coordinates is not required for the point 0,0):

- Crop the image to a rectangular region of width `W` and height `H` from the upper left corner:

`jpegtran -crop {{W}}x{{H}} -outfile {{path/to/output.jpg}} {{path/to/image.jpg}}`

**Comment by ev-john** [comment]
Thanks a lot @navarroaxel, everything updated now.

**Review by owenvoke — APPROVED** [review]
LGTM!

File: pages/linux/jpegtran.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Transverse the image across the upper right to lower left axis:
+
+`jpegtran -transverse {{path/to/image.jpg}} > {{path/to/output.jpg}}`
+
+- Convert the image to grayscale:
+
+`jpegtran -grayscale {{path/to/image.jpg}} > {{path/to/output.jpg}}`
+
+- Crop the image to a rectangular region of width `W` and height `H` from the upper left corner:
```
Comment (navarroaxel) [inline pages/linux/jpegtran.md:26]:
```suggestion
- Crop the image to a rectangular region of width `W` and height `H` from the upper left corner, saving the output to a specific file:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @ev-john, just a clarification.

**Review by navarroaxel — APPROVED** [review]

**Comment by ev-john** [comment]
Thank you @navarroaxel @owenvoke :)

**Pull request merged**